v1.0/porting-postgres.md

```diff
@@ -96,3 +96,36 @@ In CockroachDB, no such modulo is performed.
 ~~~ sql
 SELECT 1::int << (x % 64)
 ~~~
+
+### Casting `BYTES`/`BYTEA` to `STRING`/`TEXT`
+
+In PostgreSQL, casting a byte array to a string results in the escaped byte
+string consisting of each hexadecimal digit:
+
+~~~ sql
+postgres=# SELECT '\x48656c6c6f20776f726c6421'::BYTEA::TEXT;
+            text
+----------------------------
+ \x48656c6c6f20776f726c6421
+~~~
+
+In CockroachDB, this conversion is performed by attempting to convert the
+byte data directly into a string:
+
+~~~ sql
+SELECT '\x48656c6c6f20776f726c6421'::BYTEA::TEXT;
++---------------------------------------------+
```

Comment:
nit: In cockroach examples, we tend to break the command and the response into separate code blocks.